pkg/cloud/cloud.go

```diff
@@ -347,7 +383,7 @@ func (c *cloud) AttachDisk(ctx context.Context, volumeID, nodeID string) (string
 
 	if !device.IsAlreadyAssigned {
 		request := &ec2.AttachVolumeInput{
-			Device:     aws.String(device.Path),
+			Device:     aws.String("disk2"),
```

Comment:
Is it expected here?

Reply:
This field means nothing for c2 api, but has to be in format "disk{number}".